api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2190,6 +2190,27 @@ message UpsertTokenV2Request {
   }
 }
 
+// Request for GetHeadlessAuthentication.
+message GetHeadlessAuthenticationRequest {
+  // Name is the headless authentication id.
```

Comment:
If the name is headless authentication id why this field is not called `AuthenticationID`? 😅